Assets/Project/Modules/Enemies/Slime/Scripts/SlimeRefactor/SlimeHealth.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SlimeHealth : ISlimeComponent,IDamageHitTarget
```

Comment:
Again, rename this class to have a common name for all enemies so we can reuse it.